sources/packages/api/src/route-controllers/application/application.controller.ts

```diff
@@ -86,13 +88,26 @@ export class ApplicationController extends BaseController {
         payload.associatedFiles,
       );
     }
-
+    // TODO:remove the static program year and add dynamic year, from program year table
```

Comment:
I would recommend moving this logic inside the ApplicationService to keep the controller cleaner.